apps/mobile/src/screens/ResultDetailScreen.js

```diff
@@ -0,0 +1,21 @@
+// @flow
+
+import * as React from 'react';
+import { ResultDetail } from '@kiwicom/margarita-core';
+
+type Props = {|
+  +navigation: {|
```

Comment:
Can you use [this](https://github.com/kiwicom/margarita/blob/master/apps/mobile/src/screens/BookingDetailScreen.js#L15) type instead?